llvm/lib/Target/RISCV/RISCVInstrFormatsCOREV.td

```diff
@@ -128,7 +128,7 @@ class RVInstAlu_rri<bits<2> funct2, bits<3> funct3, dag outs, dag ins,
   let Inst{19-15} = rs1;
   let Inst{14-12} = funct3;
   let Inst{11-7} = rd;
-  let Opcode = OPC_CUSTOM2.Value;
+  let Opcode = OPC_CUSTOM1.Value;
```

Comment:
It took a while for me to understand that this is referring to the encoding of the RISC-V instructions, specifically that these instructions now sit in the _custom1_ space, not the _custom2_ space.